helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -257,18 +308,22 @@ public void run() {
           }
           try {
             logger.info(
-                "Num callbacks merged for path:" + handler.getPath() + " : " + mergedCallbacks);
-            handler.invoke(notificationToProcess);
+                "Num callbacks merged for path:" + _handler.getPath() + " : " + mergedCallbacks);
```

Comment:
this will no longer be more than 1 right, since we dedup it when we insert the event into the queue?